org.eclipse.jdt.core.tests.model/src/org/eclipse/jdt/core/tests/model/JavaSearchBugs9Tests.java

```diff
@@ -4747,5 +4754,52 @@ public void testBug547095_type_patter_search_modular() throws Exception {
 	}
 }
 
+/*
+ * Test for call hierarchy of a field with a type parameter coming from a JRE module.
```

Comment:
Note: the test **example** doesn't match the comment and method name. This is extremely confusing.